src/cli/peakdetector/PeakDetectorCLI.cpp

```diff
@@ -639,6 +685,47 @@ void PeakDetectorCLI::saveJson(string setName) {
 	}
 }
 
+QString PeakDetectorCLI::UploadToPolly(QString jsPath,QString nodePath,QStringList filenames) {
```

Comment:
@Ky5595 The whole idea was not to write a lot of logic in the CLI or GUI of El-MAVEN and have a separate library which takes care of that. I believe this function (uploadToPolly) should be part of the library .